package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "gamma-sky",
-  "version": "0.0.1",
+  "version": "0.0.0",
   "license": "MIT",
   "angular-cli": {},
   "scripts": {
-    "start": "ng server",
+    "start": "ng serve",
     "postinstall": "typings install",
     "lint": "tslint \"src/**/*.ts\"",
-    "format": "clang-format -i -style=file --glob=src/**/*.ts",
+    "test": "ng test",
```

Comment:
Maybe try running ng test also on travis-ci.
If it fails, OK to not run it this way.